Client/Renderer.cpp

```diff
@@ -602,6 +601,8 @@ void GameTechRenderer::RenderSkybox() {
 }
 
 void GameTechRenderer::RenderCamera() {
+    glEnable(GL_BLEND);
+    glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
```

Comment:
Deferred rendering doesn't support alpha blending not sure why this has been added.

Reply:
Removed. Added them before deferred to check some stuff if i had to guess. Not entirely sure however.